django_toolkit/middlewares.py

```diff
@@ -11,3 +17,27 @@ class VersionHeaderMiddleware(object):
     def process_response(self, request, response):
         response['X-API-Version'] = settings.TOOLKIT['API_VERSION']
         return response
+
+
+class AccessLogMiddleware(object):
+
+    LOG_FORMAT = (
+        u'[{user}] '
+        u'{response.status_code} {request.method} {request.path}'
+    )
```

Comment:
It would be nice to have this `LOG_FORMAT` imported from the settings.

Note: The PR #16 is introducing a module for this project's settings